### ISSUE TITLE
DISPLAY: Change style of x button

### ISSUE THREAD

**Issue by jjoteal** [18F/college-choice#1031]
I know this was a bug somewhere, but I can not for the life of me find it.

Change the "x" button style from this (current site screenshot):

![image](https://cloud.githubusercontent.com/assets/12192706/9691401/35091530-5310-11e5-8c02-2cc2730b7aca.png)

To what was designed:

![image](https://cloud.githubusercontent.com/assets/12192706/9691435/70b5344c-5310-11e5-9088-c7c170036ad3.png)

**Comment by LisaGee** [comment]
I made it post-launch b/c I think it’s functionality needs to be researched/addressed.

https://github.com/18F/college-choice/issues/426

The functionality works as it is currently implemented. And what if the user clicked Add State,b ut then whoops, decides she doesn't really want to add another state and wants the empty field to go away?

**Comment by jjoteal** [comment]
Just adding my comment from the other ticket here:
This is JUST referring to the style of the 'x' -- the circle should be smaller, the 'x' should be white, and the 'x' within the circle should be bigger.

**Comment by meiqimichelle** [comment]
Looks like:
![screen shot 2015-09-05 at 2 56 43 pm](https://cloud.githubusercontent.com/assets/4827522/9701447/6404043e-53de-11e5-8394-1ca711d3173d.png)

I also fixed some of the style issues on the 'Add State' in passing (tightened up the line height, and I think got rid of that bug where the edge would be cut off on certain browsers). I'm not sure why, when the add state and x are on the same line, that the add state drops too low on the line, but since we really don't want those together (#426), I'm not going to try to fix it.

Also, I ended up using a font awesome icon for the circle-x because of the problems we've had in cross-browser getting our accordion x'es to display centered and lovely. That is why our vision looks a little chunkier than the mock -- I don't have control over the weight of the x. However, I think it's a fair tradeoff.
